smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/member/MemberShapeDecodeGenerator.kt

```diff
@@ -168,6 +205,9 @@ open class MemberShapeDecodeGenerator(
                 is ListShape, is SetShape -> " ?? []"
                 // Maps can only have empty map as default value
                 is MapShape -> " ?? [:]"
+                is TimestampShape -> " ?? Date(timeIntervalSince1970: ${it.expectNumberNode().value})"
+                is DocumentShape -> { resolveDocumentDefault(false, it) }
+                is BlobShape -> { resolveBlobDefault(targetShape, it.toString()) }
```

Comment:
nit: only want to put scope if its multi-line, can remove

Reply:
Change adopted